### PR TITLE
HW7_Hanna_Pidchasiuk

### PR DESCRIPTION
**Pull request by pidchasiuk**
Left in the first task first realization of the function. Interesting if it is good to do like this, or better choose other solutions.

For task 4 - can't find better solutions rather than first open file in read mode, and then write. In case I try "r+" my new lines are additionally added, but not overwritten, and if I try to truncate file it adds unknown symbols to the beginning file. And in case "w+" it clears file info before I'm getting lines from it.

### CODE REVIEW

File: HannaPidchasiuk/HW7/task4.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# You have a file of unknown length.
+# Write a function that will remove all numbers from each line of the file.
+import re
+
+
+def remove_numbers(lines: list) -> list:
```
Comment (bohdandovbsyh) [inline HannaPidchasiuk/HW7/task4.py]: it is better to have file_path as a parameter, and read file inside the function

File: HannaPidchasiuk/HW7/task4.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        result.append(re.sub(r'[0-9]', '', line))
+    return result
+
+
+if __name__ == '__main__':
+    with open("text.txt", "r") as file:
+        lines_list = remove_numbers(file.readlines())
+
+    with open("text.txt", "w") as file:
+        file.writelines(lines_list)
```
Comment (bohdandovbsyh) [inline HannaPidchasiuk/HW7/task4.py]: it is not necessary for this task to write it back. But ok.

File: HannaPidchasiuk/HW7/task1.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#     actions = {
+#         "+": left_operand + right_operand,
+#         "-": left_operand - right_operand,
+#         "*": left_operand * right_operand,
+#         "/": left_operand / right_operand
+#     }
+#     if actions.get(operation) is None:
+#         return f'Not known operation: {operation}'
+#     else:
+#         return actions.get(operation)
```
Comment (bohdandovbsyh) [inline HannaPidchasiuk/HW7/task1.py:17]: it is an even better solution actually.

**Review by bohdandovbsyh — APPROVED** [review]

File: HannaPidchasiuk/HW7/task4.py (hunk 1 above)
Comment (pidchasiuk) [reply HannaPidchasiuk/HW7/task4.py]: fixed

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**